packages/blockchain-wallet-v4-frontend/src/scenes/Signup/components/Header/index.tsx

```diff
@@ -36,20 +38,31 @@ const SubHeader = styled(Text)`
   `}
 `
 
-const Header = () => (
+const Header = (props: SubviewProps) => (
   <Wrapper>
     <Column>
       <HeaderText color='whiteFade900' weight={700} data-e2e='signupSecureHeader'>
-        <FormattedMessage
-          id='scenes.register.securelybuy'
-          defaultMessage='Securely Buy, Sell, and Store Crypto.'
-        />
+        {!props.isLatam && (
+          <FormattedMessage
+            id='scenes.register.securelybuy'
+            defaultMessage='Securely Buy, Sell, and Store Crypto.'
+          />
+        )}
+        {props.isLatam && (
+          `Envía, recibe y obtén hasta un 14g% de ganancias con tus criptos.`
```

Comment:
why is this not translated?

Reply:
I am waiting to have access to lokalize. But this latam signup page will be in production on April 18 is everything goes ok.

Reply:
@schnogz could you give him access to lokalize?